CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Change Log
 
 ## master (unreleased)
-
+- add support for api extraction from statically linked libraries.
```

Comment:
perhaps "extract function and API names from ELF symtab entries @yelhamer #1443"